common-content/en/module/decomposition/frontend-limitations-intro/index.md

```diff
@@ -0,0 +1,36 @@
++++
+title = "Limitations of frontends"
+headless = true
+time = 5
+facilitation = false
+emoji= "📖"
+objectives = [
+    "Explain what can't be achieved with just a frontend",
+]
++++
+
+You've made static websites, both as part of the "world-wide web" reading, and in previous courses/modules.
+
+These were HTML pages, which perhaps included some JavaScript. The JavaScript made them interactive. But they were still static.
+
+A static website is one where your web browser requests some files from a server, and all the web server does is hand over the files.
+
+From that point on, your web browser works on its own. Any event handlers, timers, etc are processed by the web browser.
+
+And the job of the server was just to hand over files. Those files will probably be the same, no matter what user asked for them, or what computer they were using.
+
+We often call these static files a **frontend**.
+
+And we can build impressive, useful websites with just a frontend. We can include images, videos, and sounds. We can react to user input. We can change what we show on the page. We can even use things like [the Web Storage API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API) to store data so that if you leave the page and come back, your data is still there.
```

Comment:
```suggestion
We can build impressive, useful websites with just a frontend. We can include images, videos, and sounds. We can react to user input. We can change what we show on the page. We can even use things like [the Web Storage API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API) to store data so that if you leave the page and come back, your data is still there.
```

You can also say that the curriculum website is a static site. I would usually say that Smashing Magazine is, but I don't think you're using exactly the same definition here.